### PR TITLE
Support --no-sudo with --install-deps

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

**Comment by ekohl** [comment]
Though it's order dependant, :+1:

**Comment by domcleal** [comment]
Yes, I noticed that.  I can extract that yum install command into a separate method if you'd prefer.

**Comment by ekohl** [comment]
That would be much cleaner.

File: virt-spawn [hunk 1]
```diff
@@ -105,6 +105,7 @@ Example:
 USAGE
 }
 
+ACTION=build
```
Comment (domcleal) [inline virt-spawn]: Using a single ACTION variable instead of one per action to hopefully keep it tidier if more actions are added.

**Comment by domcleal** [comment]
Indeed, updated with that code refactored out.

**Comment by ekohl** [comment]
:+1:

**Comment by mmoll** [comment]
merged, thanks @domcleal!

**Pull request merged**